changelog/as-fix-ece-variable-subs

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: dev
+
+Refine verification for disabling ECE on subscriptions that require shipping.
```

Comment:
I'm adding the CHANGELOG entry for now, but it will ultimately be removed when I create the next PR to address #9771.